src/components/VideoPopoverMenu/index.tsx

```diff
@@ -1,29 +1,26 @@
-import PropTypes from 'prop-types';
-import React from 'react';
+import React, {useRef} from 'react';
+import type {View} from 'react-native';
+import type {PopoverMenuItem} from '@components/PopoverMenu';
 import PopoverMenu from '@components/PopoverMenu';
 import {useVideoPopoverMenuContext} from '@components/VideoPlayerContexts/VideoPopoverMenuContext';
+import type {AnchorPosition} from '@styles/index';
 
-const propTypes = {
-    isPopoverVisible: PropTypes.bool,
-
-    hidePopover: PropTypes.func,
-
-    anchorPosition: PropTypes.shape({
-        horizontal: PropTypes.number.isRequired,
-        vertical: PropTypes.number.isRequired,
-    }),
+type VideoPopoverMenuProps = {
+    isPopoverVisible: boolean;
+    hidePopover: (selectedItem?: PopoverMenuItem, index?: number) => void;
+    anchorPosition: AnchorPosition;
```

Comment:
```suggestion
    isPopoverVisible?: boolean;
    hidePopover?: (selectedItem?: PopoverMenuItem, index?: number) => void;
    anchorPosition?: AnchorPosition;
```

According to propTypes

Reply:
Same as [here](https://github.com/Expensify/App/pull/37425#discussion_r1526119502)